eternaleconomy-core/src/main/java/com/eternalcode/economy/leaderboard/LeaderboardCommand.java

```diff
@@ -0,0 +1,83 @@
+package com.eternalcode.economy.leaderboard;
+
+import com.eternalcode.economy.EconomyPermissionConstant;
+import com.eternalcode.economy.account.Account;
+import com.eternalcode.economy.account.AccountManager;
+import com.eternalcode.economy.config.implementation.PluginConfig;
+import com.eternalcode.economy.format.DecimalFormatter;
+import com.eternalcode.economy.format.DurationFormatter;
+import com.eternalcode.economy.multification.NoticeService;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Command(name = "leaderboard")
```

Comment:
The command name is too generic, please use `/balancetop` as the main route, and in alias entries `/baltop` `/balanceleaderboard`